### PR TITLE
CONSOLE-3818: Remove Chrome from the console builder image

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Follow on after code review: Build a new image based on this change, and push to quay.io,  then verify in CI before merging the change.

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm

File: Dockerfile [hunk 1]
```diff
@@ -41,9 +41,8 @@ RUN test -d ${REMOTE_SOURCES}/cachito-gomod-with-deps || exit 0; \
     cp -f $REMOTE_SOURCES_DIR/cachito-gomod-with-deps/app/registry-ca.pem . \
  && cp -f $REMOTE_SOURCES_DIR/cachito-gomod-with-deps/app/frontend/{.npmrc,.yarnrc,yarn.lock} frontend/
 
-# prevent download of chromedriver, geckodriver, and the cypress binary as part of module installs
-ENV CHROMEDRIVER_SKIP_DOWNLOAD=true \
-    GECKODRIVER_SKIP_DOWNLOAD=true \
+# prevent download of geckodriver, and the cypress binary as part of module installs
+ENV GECKODRIVER_SKIP_DOWNLOAD=true \
```
Comment (rhamilto) [inline Dockerfile:45]: I believe `GECKODRIVER_SKIP_DOWNLOAD=true` can be removed as well as that applied to Protractor.

File: frontend/package.json
Comment (rhamilto) [inline frontend/package.json:1]: I believe `"geckodriver": "1.x",` can be removed as well as that applied to Protractor.

**Comment by cyril-ui-developer** [comment]
FYI: @bipuladh @vikram-raj  @ciiay @lokanandaprabhu @tnisan @jerolimov @Lucifergene 
Console will be removing Chrome dependency with this PR change. Please update your package(s) if this change impacts it.

**Comment by jhadvig** [comment]
Adding labels manually since we are removing unused package.

**Pull request merged**